test/tizen_capi/unittest_tizen_capi_layer.cpp

```diff
@@ -87,9 +87,69 @@ TEST(nntrainer_capi_nnlayer, setproperty_02_p) {
 
   status = ml_nnlayer_set_property(handle, ML_BIAS_ZERO, "true");
   EXPECT_EQ(status, ML_ERROR_NONE);
-  
+
+  status = ml_nnlayer_set_property(handle, ML_ACTIVATION, "sigmoid");
+  EXPECT_EQ(status, ML_ERROR_NONE);
+}
+
+/**
+ * @brief Neural Network Layer Create / Delete Test (negative test )
+ */
+TEST(nntrainer_capi_nnlayer, setproperty_03_n) {
+  ml_nnlayer_h handle;
+  int status;
+  status = ml_nnlayer_create(&handle, ML_LAYER_TYPE_INPUT);
+  EXPECT_EQ(status, ML_ERROR_NONE);
   status = ml_nnlayer_set_property(handle, ML_ACTIVATION, "sigmoid");
-  EXPECT_EQ(status, ML_ERROR_NONE);    
+  EXPECT_EQ(status, ML_ERROR_INVALID_PARAMETER);
```

Comment:
I think even if this is a negative case, you may need to destroy the created instance.

Reply:
Ooops. Thanks. fixed.